[2022-08-29 00:00:02,545] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29 00:00:02,553] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29 00:00:02,553] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 00:00:02,553] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-29 00:00:02,554] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 00:00:02,568] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-28 00:00:00+00:00
[2022-08-29 00:00:02,572] {standard_task_runner.py:52} INFO - Started process 10679 to run task
[2022-08-29 00:00:02,575] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-28T00:00:00+00:00', '--job-id', '272', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmp80qzcdee', '--error-file', '/tmp/tmpuumaqeqh']
[2022-08-29 00:00:02,575] {standard_task_runner.py:80} INFO - Job 272: Subtask link_dag
[2022-08-29 00:00:02,625] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-28T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-29 00:00:02,688] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-28T00:00:00+00:00
[2022-08-29 00:00:44,450] {python.py:173} INFO - Done. Returned value was: None
[2022-08-29 00:00:44,462] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220828T000000, start_date=20220829T000002, end_date=20220829T000044
[2022-08-29 00:00:44,514] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29 00:00:44,542] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
